### DISCUSSION TITLE
Choppy/laggy video in HK

### DISCUSSION THREAD

**Discussion by fentonmatt** [dgreif/ring#814] [General]
@dgreif is there a way to adjust the video feed settings for the Ring cameras? To improve how they stream to HK? I have done everything possible to improve my streaming to HK for the cameras and it consistently isn't reliable. I have a RPi 4 Model B with Homebridge and am hardwired. I never have any issues through the Ring app and this happens when I am home and away. Just super choppy and laggy. I'd like to use the new ATV feature where it shows motion or rings on the screen but with unreliable video, that is useless. Any suggestions to improve my setup?

**Comment by dgreif** [comment]
> I never have any issues through the Ring app and this happens when I am home and away

This is likely just a side effect of all the network hops required for Ring + HomeKit.  See https://github.com/dgreif/ring/wiki/Camera-Troubleshooting#live-stream-quality
